components/nordigen/sources/new-transaction/new-transaction.mjs

```diff
@@ -134,19 +121,34 @@ export default {
       return;
     }
 
-    const account = await this.getAccount(requisitionId);
-    if (!account) {
+    const accountId = await this.getAccount(requisitionId);
+    if (!accountId) {
       return;
     }
-    console.log("Account found: " + account);
+    console.log("Account found: " + accountId);
+
+    let lastTransactionDate = this._getLastTransactionDate();
+    const args = lastTransactionDate
+      ? {
+        params: {
+          date_from: lastTransactionDate,
+        },
+      }
+      : {};
+    const transactions = await this.nordigen.listTransactions(accountId, args);
 
-    const transactions = await this.listTransactions(account);
     transactions.forEach((transaction) => {
+      if (!lastTransactionDate
+        || Date.parse(transaction.bookingDate) > Date.parse(lastTransactionDate)) {
+        lastTransactionDate = transaction.bookingDate;
+      }
       this.$emit(transaction, {
-        summary: transaction.remittanceInformationUnstructuredArray[0],
+        summary: transaction.remittanceInformationUnstructured,
```

Comment:
@michelle0927 `summary: transaction.remittanceInformationUnstructured` must be `summary: transaction.remittanceInformationUnstructuredArray[0]`.

Because of this, the summary events are empty.

Reply:
@LucBerge Thanks for commenting. I created [this PR](https://github.com/PipedreamHQ/pipedream/issues/5472) to update the summary of `new-transaction.mjs`.